src/mock/mock_kv.rs

```diff
@@ -0,0 +1,45 @@
+// Copyright 2019 TiKV Project Authors. Licensed under Apache-2.0.
```

Comment:
Can you remove the changes to src/mock? I think they are orthogonal to the mock-tikv changes.